### ISSUE TITLE
Crash at startup (Fabric 1.17.1)

### ISSUE THREAD

**Issue by maicol07** [MoriyaShiine/bewitchment#176]
Error:

```
java.lang.IndexOutOfBoundsException: Index 2 out of bounds for length 2
```

Crash report: [bytebin.lucko.me/ORapFb9xQ6](https://bytebin.lucko.me/ORapFb9xQ6)

Crash log: [bytebin.lucko.me/ceAgTFapPq](https://bytebin.lucko.me/ceAgTFapPq)

**Comment by MoriyaShiine** [comment]
Can you be more specific? When does this happen?

**Comment by maicol07** [comment]
@MoriyaShiine when you start the game, before it opens the menu window

**Comment by MoriyaShiine** [comment]
Can you binary search (disable half of your mods until it stops crashing) and figure out which one is causing bewitchment to crash? You have a lot installed

**Comment by HitsukiDreamy** [comment]
I'm having the same problem as well, and it seems to come down to the PlayerAbilityLib mod, and neither version works. I've tested every mod that Bewitchment relies on, including the main mod itself, but every time PAL is added, the game crashes without the Fabric Loader giving its usual error messages. The only other additional mods I have outside of these are Iris and Illuminations, I have tested those as well, and they are not the cause.
